src/views/Home/Home.tsx

```diff
@@ -1,70 +1,84 @@
 
 import NavBar from "components/organisms/NavBar/NavBar";
-import TaskContainer from "components/organisms/Containers/TaskContainer/TaskContainer";
 import SideBar from "components/organisms/SideBar/SideBar";
 import Modals from "components/organisms/Modals/Modals";
+import FilteredContainer from "components/organisms/Containers/FilteredContainer/FilteredContainer";
 import CalendarContainer from "components/organisms/Containers/CalendarContainer/CalendarContainer";
 import TagsContainer from "components/organisms/Containers/TagsContainer/TagsContainer";
-import FilteredContainer from "components/organisms/Containers/FilteredContainer/TaskContainer";
-
-import { fetchListArray, selectActiveListID } from "features/list/listSlice";
-import { setEmail, setUserName } from "features/user/userSlice";
-import { fetchTags } from "features/tag/tagSlice";
+import TaskContainer from "components/organisms/Containers/TaskContainer/TaskContainer";
 
+import { getTodayString, getTomorrowString } from "utils/helpers";
 import * as S from "./styles";
 
 const Home = () => {
+  const [isFetching, setIsFetching] = useState(true);
+  const [sideBarVisible, setSideBarVisible] = useState(false);
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
-  const selectedList = useAppSelector(selectActiveListID);
-
-  const [sideBarVisible, setSideBarVisible] = useState(false);
 
   const handleToggleSideBar = () => {
     setSideBarVisible(!sideBarVisible);
   };
 
+  const tomorrowFilters = {
+    date: getTomorrowString(),
+  };
+
+  const todayFilters = {
+    date: getTodayString(),
+  };
+
   useEffect(() => {
+    setIsFetching(true);
     const token = localStorage.getItem("jwt");
     if (token) {
       const decoded: { userId: string; email: string; userName: string } =
         jwt_decode(token);
-      dispatch(fetchListArray());
-      dispatch(fetchTags());
       dispatch(setUserName(decoded.userName));
       dispatch(setEmail(decoded.email));
+      dispatch(fetchTags());
+
+      dispatch(fetchListArray()).then(() => {
+        setIsFetching(false);
+      });
     } else {
       navigate("/login");
     }
-  }, [dispatch, navigate]);
-
-  const container = () => {
-    switch (selectedList) {
-      case "Filtered":
-      case "Today":
-      case "Tomorrow":
-        return <FilteredContainer />;
-      case "Calendar":
-        return <CalendarContainer />;
-      case "Tags and filters":
-        return <TagsContainer />;
-      default:
-        return <TaskContainer />;
-    }
-  };
+    //eslint-disable-next-line
+  }, []);
 
   return (
     <div className="w-full ">
       <NavBar handleToggleSideBar={handleToggleSideBar} />
       <div className="w-full mt-14 ">
         <SideBar isVisible={sideBarVisible} />
-        <S.MainContainer $isVisible={sideBarVisible}>
-          {container()}
-        </S.MainContainer>
+        {!isFetching && (
+          <S.MainContainer $isVisible={sideBarVisible}>
+            <Routes>
```

Comment:
why you have routers in two places?

Reply:
router is only in app.tsx, route definitions are split in two places, because they were getting a bit long. Home.tsx routes change the main task container in Home. Should I move them all to one place?

Reply:
ok - nie jestem pewien co routów dawno nie miałem w aplikacji - możesz zostawić na razie